src/likelihoodratiotest.jl

```diff
@@ -26,6 +26,29 @@ function Base.propertynames(lrt::LikelihoodRatioTest, private::Bool=false)
     return (:deviance, :formulas, :models, :pvalues, :tests)
 end
 
+"""
+    StatsAPI.pvalue(lrt::LikelihoodRatioTest)
```

Comment:
```suggestion
    pvalue(lrt::LikelihoodRatioTest)
```
The qualification in the docstring seems to suggest that it's how the user should expect to call the function, but StatsAPI is not intended to be directly user-facing. Since we're also re-exporting `pvalue`, the StatsAPI bit is irrelevant anyway.